src/pages/en/guides/content-collections.md

```diff
@@ -32,15 +33,52 @@ export default defineConfig({
 });
 ```
 
-You will also need to update `tsconfig.json`. Add `"strictNullChecks": true` under `compilerOptions`.
+### Update TypeScript configuration
 
-```json title="tsconfig.json" ins={3}
-{
-  "compilerOptions": {
-    "strictNullChecks": true
+To benefit from the full TypeScript and autocompletion features of content collections, you may also need to update `tsconfig.json`. Proper TypeScript configuration is required to [use schemas with your collections](#defining-a-collection-schema).
+
+To enable the setting necessary for schemas, follow the instructions below based on your Astro project's current [TypeScript configuration](/en/guides/typescript/#setup). 
+
+<TypeScriptSettingTabs>
+  <Fragment slot="relaxed">
+  If you are using Astro's `base` setting (e.g. you set up your project with "relaxed" TypeScript), add `"strictNullChecks": true` under `compilerOptions`. 
+  
+  Or, you can change your `base` TypeScript setting to `strict` or `strictest` to use TypeScript's features throughout your entire Astro project.
```

Comment:
This might suggest the `strictNullChecks` option _doesn’t_ apply throughout your project, so maybe safer to say something like this?

```suggestion
  Or, you can change your `base` TypeScript config to `strict` or `strictest` to use a broader set of stricter type-checks in your project.
```